docs/nuget-org/policies/deleting-packages.md

```diff
@@ -34,3 +34,25 @@ Packages that meet any of the following criteria are not allowed on the public N
 If you find a package that is in violation of any of these items, click the **Report Abuse** link on the package details page and submit a report.
 
 Note that the NuGet team and the .NET Foundation reserves the right to change these criteria at any time.
+
+## Unlisting a package
+The purpose of unlisting a package version is that it hides the package version from search and from nuget.org package details page. Furthermore it allows existing users of the package to keep downloading it, but reduces new adoption since the package is not visible in search.
+
+In order to unlist a specific package version please follow these steps:
+
+** Unlist a specific package version **
+- Click the `Account name` at the top right corner. 
+- Click `Manage packages`
+- Click `Published packages`
+- Click the package name which version you want to unlist
+You will now see all the versions on that package. 
+- Under the `Status` column click the `Listed` link on that package version you want to unlist.
+- You will now be transferred to the "Manage" page of that specific nuget package version. The "Manage" page will display following sections: "Owners", "Deprecation", "Listing" and "Documentation".
```

Comment:
If possible, let's avoid mentioned all the current sections since they may change over time making this doc out of date or even confusing. We should be able to drop this step and change the next step from `... beside "Listing"...` to `... beside the "Listing" section...`